articles/tables.md

```diff
@@ -309,16 +314,17 @@ syn.store(Table(schema, df))
 {% tab R %}
 {% highlight r %}
 # Define a new column
-newColumn <- TableColumn(name="purchased", columnType="STRING")
+newColumn <- synStore(Column(name="purchased", columnType="STRING"))
 # Add the new column to existing schema
-schema <- synAddColumn(schema, newColumn)
+schema <- schema$addColumn(newColumn)
 schema <- synStore(schema)
 # Query for the newest schema
 queryResult <- synTableQuery('select * from syn7264701')
+df <- as.data.frame(queryResult)
 # Add values
-queryResult@values['purchased'] <- c('yes', 'yes', 'no', 'yes') 
+df['purchased'] <- c('yes', 'yes', 'no', 'yes') 
```

Comment:
should be `df$purchased` or `df[, 'purchased']`

Reply:
check internally consistent how we refer to columns in a data frame; then determine how to be consistent w/ vignettes.